drivers/adc-dac/ad5592r/ad5593r.c

```diff
@@ -50,6 +50,7 @@
 
 #define STOP_BIT	1
```

Comment:
we use (or at least try to) a bunch of tabs to align the define values, it would be nice if they'd be aligned